mate-panel/panel-background.c

```diff
@@ -763,6 +763,7 @@ panel_background_free (PanelBackground *background)
 	background->default_pattern = NULL;
 }
 
+#ifdef HAVE_X11
```

Comment:
What's that about (ant the other disabling of the background stuff)? Couldn't this just disable the problematic XLib piece, and still support at least color?